### PR TITLE
UG and DG changes for Goal Feature and Le's PPP

### PR DESCRIPTION
**Pull request by mgiang2015**
UG:
- Add valid page description.

DG:
- Add extensions for Goal Command use case (UC08)
- Add use case for GoalDel command (UC11)

PPP:
- Create PPP file and fill in some info
- Change link to PPP file in AboutUs.md

### CODE REVIEW

**Review by TanLeYang — APPROVED** [review]
LGTM!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -419,6 +419,7 @@ you can motivate yourself to read.
    * The `INDEX` **must be a positive number** 1, 2, 3, …
    * If you add a goal to a book with an existing goal, the newly added goal will replace it.
    * You can only set dates that have not passed
+   * You can only set pages higher than your current bookmarked page, and lower than total pages of the book.
```
Comment (angrybunny123) [inline docs/UserGuide.md]: Maybe this should be "Lower than or equal to the total pages of the book"

**Review by angrybunny123 — COMMENTED** [review]
LGTM besides the one comment that I made

**Review by pennhanlee — APPROVED** [review]
LGTM

**Pull request merged**